addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -97,7 +97,10 @@
             <field name="arch" type="xml">
                 <xpath expr="/form/*" position="before">
                     <header>
-                        <button name="action_submit_expenses" states="draft" string="Submit to Manager" type="object" class="oe_highlight"/>
+                        <widget name="attach_document" string="Attach Document" action="send_attach_document" attrs="{'invisible': [('attachment_number', '&lt;=', 0)]}"/>
+                        <widget name="attach_document" string="Attach Document" action="send_attach_document" class="oe_highlight" attrs="{'invisible': [('attachment_number', '&gt;=', 1)]}"/>
```

Comment:
I would use the YAGNI principle and set ``action="message_post"`` instead. 

If one day we want to make something more specific, we could use a method instead ;)